static/admin/config.yml

```diff
@@ -28,6 +28,20 @@ collections:
               default: "home-page",
             }
           - { label: "Path", name: "path", widget: "hidden", default: "/" }
+          - {
```

Comment:
non blocking,  but let's find a solution to avoid repeated code blocks in the yml file. I will try to find a solution today